manifest.yml

```diff
@@ -0,0 +1,15 @@
+---
+declared-services:
+  my-conversation-service:
+    label: conversation
+    plan: free
+applications:
+- name: watson-conversation-slots-intro
+  command: npm start
+  path: .
+  memory: 256M
+  instances: 1
+  services:
+  - my-slots-intro-conversation
```

Comment:
here I've been using a project prefix. There is wos-conversation, wbc-conversation etc. You can have wcsi or just csi if you want it to be cool. I think we often use -services on the end (seems redundant tho and don't know if we have been consistent with that).  "my-" might be consistent with some of the SDK examples but I think it would be better to distinguish from that.